static/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidates.tsx

```diff
@@ -130,47 +133,60 @@ class Candidates extends Component<Props, State> {
   doSearch = debounce(this.filterCandidatesBySearch, 300);
 
   getFilters() {
+    const {imageStatus} = this.props;
     const candidates = [...this.props.candidates];
     const filterOptions = this.getFilterOptions(candidates);
+
+    const defaultFilterSelections = (
+      filterOptions.find(section => section.value === 'status')?.options ?? []
+    ).filter(
+      opt =>
+        opt.value !== `status-${CandidateDownloadStatus.NOT_FOUND}` ||
+        imageStatus === ImageStatus.MISSING
+    );
+
     this.setState({
       filterOptions,
+      filterSelections: defaultFilterSelections,
       filteredCandidatesBySearch: candidates,
       filteredCandidatesByFilter: this.getFilteredCandidatedByFilter(
         candidates,
-        filterOptions
+        defaultFilterSelections
       ),
     });
   }
 
   getFilterOptions(candidates: ImageCandidates) {
-    const {imageStatus} = this.props;
-
-    const filterOptions = {};
+    const filterOptions: FilterOptions = [];
 
     const candidateStatus = [
       ...new Set(candidates.map(candidate => candidate.download.status)),
     ];
 
     if (candidateStatus.length > 1) {
-      filterOptions[filterOptionCategories.status] = candidateStatus.map(status => ({
-        id: status,
-        symbol: <Status status={status} />,
-        isChecked:
-          status !== CandidateDownloadStatus.NOT_FOUND ||
-          imageStatus === ImageStatus.MISSING,
-      }));
+      filterOptions.push({
+        value: 'status',
+        label: filterOptionCategories.status,
+        options: candidateStatus.map(status => ({
+          value: `status-${status}`,
```

Comment:
do we need a prefix?

Reply:
Added prefixes to ensure that there's no conflict between `status` and `source` options  – we could conceivably have a status with the same name as a source, breaking the uniqueness requirement among select option values